### DISCUSSION TITLE
Cover art from Spotify in different playlists

### DISCUSSION THREAD

**Discussion by xaviugs** [BimmerGestalt/AAIdrive#334] [General]
Hi Hufman,

Firts of all, I'd like to great you for your job.

When I listen to music in my ID5 spotify, i noticed that when I chose a song from an artist the cover does not appear on the ID5 display, but it appears when I chose a song from a playlist. Why it happens?

Thanks a lot.

**Comment by hufman** [comment]
Thank you for the kind words!
The coverart is provided directly from Spotify, so I can't confidently say why it behaves in different ways. The playlist might have the coverart cached, while the artist radio might not be cached, but I'm just guessing.

**Comment by xaviugs** [comment]
When I play music from not a playlist, the cover art appears 2 seconds and then disappeared again.

**Reply by hufman** [reply]
Did you grant access for AAI to browse your Spotify Library? This additionally enables an API connection to fetch the cover art more reliably, while the other API that generically controls active music apps doesn't always successfully load Spotify's cover art.
Otherwise I can't think of anything else, and I haven't experienced it myself.
